.github/workflows/release.yaml

```diff
@@ -82,7 +87,10 @@ jobs:
         run: podman login -u ${{ secrets.BUNDLE_PUSH_USER  }} -p ${{ secrets.BUNDLE_PUSH_PASS }} quay.io
 
       - name: Create image from snapshot
-        run: make push-snapshot-image IMAGE_TAG=${{ github.sha }}
+        run: make push-snapshot-image IMAGE_TAG=$TAG IMAGE_REPO=$IMAGE_REPO
+
+      - name: Create and push the tekton bundle tagged with sha
+        run: make task-bundle TASK_TAG=$TAG TASK=<( yq e '.spec.steps[].image? = "$IMAGE_REPO:$TAG"' $TASK)
 
-      - name: Create and push the tekton bundle
-        run: make task-bundle-snapshot TASK_TAG=${{ github.sha }}
+      - name: Create and push the tekton bundle tagged with snapshot
+        run: make task-bundle TASK_TAG=snapshot TASK=<( yq e '.spec.steps[].image? = "$IMAGE_REPO:snapshot"' $TASK)
```

Comment:
I don't really think building the bundle twice is a good idea. Let's build it once and tag that image as needed.

Reply:
yeah, I was thinking making the bundle tag match the image tag, but I think having the image tag of "snapshot" is just a bad idea either way. I reverted this.